cmd/crane/cmd/append.go

```diff
@@ -78,6 +97,7 @@ func NewCmdAppend(options *[]crane.Option) *cobra.Command {
 	appendCmd.Flags().StringVarP(&newTag, "new_tag", "t", "", "Tag to apply to resulting image")
 	appendCmd.Flags().StringSliceVarP(&newLayers, "new_layer", "f", []string{}, "Path to tarball to append to image")
 	appendCmd.Flags().StringVarP(&outFile, "output", "o", "", "Path to new tarball of resulting image")
+	appendCmd.Flags().BoolVar(&annotate, "annotate", false, "If true, annotate the resulting image as being based on the base image")
```

Comment:
I know I suggested this flag name, but I'm a little worried this will get confused with `crane mutate`'s `annotation` flag... wdyt?

Reply:
`crane append -b ubuntu --set-base-image-annotations` ?

I don't mind being verbose.